techcity/events/management/commands/resolve_event_recording_names.py

```diff
@@ -0,0 +1,122 @@
+import json
+import os
+
+import requests
+from django.core.management.base import BaseCommand
+
+from techcity.events.models import Event, EventRecording
+
+
+class Command(BaseCommand):
+    help = "Attempt to resolve the names of event recordings"
```

Comment:
example run of this script,

```py
$ uv run manage.py resolve_event_recording_names

Recording: recording.title='PDF Text Extraction With Python' recording.published_at=datetime.datetime(2024, 8, 15, 3, 53, 12, tzinfo=datetime.timezone.utc)
Event: event.name='PDF Text Extraction (2nd Wed Talk)' event.start_at=datetime.datetime(2024, 8, 14, 23, 0, tzinfo=datetime.timezone.utc)
``

Reply:
This seems cool. I guess we'd only need to run this when there is a recording that doesn't have an event attached, so the actual API calls would be super low. Also, if we are pulling from the known playlists where I'd be adding recordings, there could be a pretty high degree of confidence in the matching.
 
You've probably done more OpenAI stuff than I have. How deterministic do you think this is likely to be?